src/main/java/com/urlparser/Main.java

```diff
@@ -0,0 +1,19 @@
+package com.urlparser;
+
+import com.urlparser.entity.LinksFromUrl;
+import com.urlparser.model.Links;
+
+import java.net.URL;
+
+public class Main {
+
+  public static void main(String[] args) {
+
+    String urlInput = args[0];
```

Comment:
@undsett it is manadatory to give variables simple names. in this case `url` is enough, there are no other urls. Compound name is considered a codesmell(not always, but still)
http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

Reply:
Yes, but i have 2 url variables. One that contains url input and enother used in forEach. So i need to define them properly.